test/fixtures/settings.yml

```diff
@@ -418,3 +418,13 @@ attribute98:
   default: "['/usr/bin/cat']"
   value: "['/usr/bin/cat']"
   description: "Fedora CoreOS Transpiler"
+attribute99:
+  name: password_hash
+  category: Setting::Auth
+  default: 'pbkdf2sha1'
+  description: "Password hash"
+attribute100:
+  name: pbkdf2_cost
+  category: Setting::Auth
+  default: 1000
+  description: "PBKDF2 cost"
```

Comment:
fixutures for settings are not needed anymore :)

Reply:
Like not at all? Why the file is still present?

Reply:
Removed.